docs/index.md

```diff
@@ -66,6 +64,18 @@ $ mkdocs --version
 mkdocs, version 0.15.3
 ```
 
+#### Need to install Python or Pip?
+
+Install [Python] by downloading the installer from [https://www.python.org/downloads/](https://www.python.org/downloads/).
+
+> [pip] is already installed if you're using Python 2 >=2.7.9 or Python 3 >=3.4 downloaded from python.org, but you'll need to upgrade pip.
```

Comment:
This is not a quote, so let's not put it in a blockquote. We have the [Admonition Extension][1] enabled, so you can use a "note" or "warning" to make some text stand out as we do at `docs/user-guide/custom-themes.md` among other places.

[1]: https://pythonhosted.org/Markdown/extensions/admonition.html